edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -1026,35 +978,35 @@ To submit results for a competition, you must have access to the scrambles that
 7. Upload the generated scrambles from TNoodle to Scramble Matcher. Go to the folder where you saved the scramble sets after generating scrambles and select `/Interchange/<competition-name>.json` to upload. If you generated multiple sets of scrambles, such as when generating an emergency set during the competition, you can upload multiple JSON files.
 8. Assign scrambles to scramble sets by clicking the "Automatically Assign Scrambles" button. Navigate along the left hand side for each round and drag any scramble sets which were not used during the competition to the "Available" column.
 9. When all scramble sets are assigned to rounds, return to the main page and click the "Get Results JSON" button. This downloads a JSON file which contains the complete competition results.
-10. Submit results by going to the competition website and navigating to the Upload Results page. Upload the JSON file downloaded from the last step, then click the validate button. The website will run a series of checks on the results for common issues, and provide a list of errors and warnings.
-11. Resolve all errors. Read the message provided next to the error, as it contains useful information about the problem. They can often be fixed by going back to one of the previous steps. When all errors are resolved, you will need to create and upload a new JSON file from step 4 onwards and revalidate. **Do not forget to synchronize once again in WCA Live!** Common errors are:
+10. Submit the results by going to the competition website and navigating to the Upload Results page. Upload the JSON file downloaded from the last step, then click the validate button. The website will run a series of checks on the results for common issues, and provide a list of errors and warnings.
+11. Resolve all errors. Read the message provided next to the error, as it contains useful information about the problem. They can often be fixed by repeating one of the previous steps. When all errors are resolved, you will need to create and upload a new JSON file from step 4 onwards and revalidate. **Do not forget to synchronize once again in WCA Live!** Common errors are:
```

Comment:
I think that it should still be "going back to".  "repeating" indicates that only one step would be repeated, where "going back to" indicates that you would follow the steps after, which to my understanding is the intent here.